wezterm-gui/src/termwindow/mod.rs

```diff
@@ -928,6 +928,16 @@ impl TermWindow {
                 }
                 Ok(true)
             }
+            WindowEvent::DroppedFile(path) => {
+                let pane = match self.get_active_pane_or_overlay() {
+                    Some(pane) => pane,
+                    None => return Ok(true),
+                };
+                pane.writer().write_all(path.to_string_lossy().as_bytes())?;
+                pane.writer().write_all(b" ")?;
```

Comment:
I think there are some UX considerations around this that need some thought:

* The extra space is probably undesirable if I type `"` then drag a file and then want to type the closing `"`.  The space is more desirable if dropping multiple files.
* If the dragged file has spaces or other characters that might be meaningful to the shell, it might be nice to send the escaped version of the filename. How do we know what the correct escaping syntax should be?
* If bracketed paste is enabled, maybe the application will handle escaping for us if we send the filename as a bracketed paste?
* If the filename is not valid UTF-8, is it even valid to send the lossy transform of the filename?  Might we want to skip those files?

Reply:
Thanks for reply! Let me think about what to do with them.

Reply:
Escaping is a little tricky, can we consider outputting a string enclosed in quotes?